server/tests/integration/commits.test.ts

```diff
@@ -17,11 +17,35 @@
 import request from 'supertest';
 
 import app from '../../src';
+import * as mockedCustomerAuth from '../../src/middleware/__mocks__/customer-auth';
+import * as customerAuthMiddleware from '../../src/middleware/customer-auth';
 import commitFixtures from '../fixtures/commits';
 import customerFixtures from '../fixtures/customers';
 import listingFixtures from '../fixtures/listings';
 
+// Mock middlewares
+jest.mock('../../src/middleware/customer-auth');
+
+const {
+  customerAuth,
+  restoreCustomerAuth,
+} = customerAuthMiddleware as typeof mockedCustomerAuth;
+
+// Disable customer auth mock implementation by default
+beforeAll(() => {
+  restoreCustomerAuth();
+});
+
+// Disable customer auth mock implementation by default
+beforeAll(() => {
+  restoreCustomerAuth();
```

Comment:
why same restoreCustomerAuth twice?

Reply:
@ifhuang Oops! Must have been left behind from merging and I didn't notice it! Will remove one of them!